src/core_plugins/kibana/public/discover/components/field_chooser/field_chooser.js

```diff
@@ -45,13 +45,14 @@ app.directive('discFieldChooser', function ($location, globalState, config, $rou
       const filter = $scope.filter = {
         props: [
           'type',
-          'indexed',
-          'analyzed',
+          'aggregatable',
+          'searchable',
           'missing',
           'name'
         ],
         defaults: {
-          missing: true
+          missing: true,
+          type: 'any'
```

Comment:
That's the value rendered in the UI. This is master:

![image](https://cloud.githubusercontent.com/assets/1329312/26430940/0be7c838-40a6-11e7-88d7-7e07fb454df2.png)